src/main/java/uk/gov/beis/els/service/TemplatePopulator.java

```diff
@@ -189,29 +191,35 @@ public Document getPopulatedDocument() {
 
   public <T extends AnalyticsForm & SupplierNameForm> ProcessedEnergyLabelDocument asProcessedEnergyLabel(
       ProductMetadata analyticsLabel, T form) {
-    String title = String.format(analyticsLabel.getProductFileName() + " - %s - %s",
-      form.getSupplierName(), form.getModelName());
+    String analyticsAction = String.format("%s - %s", form.getSupplierName(), form.getModelName());
+    String title = String.format(analyticsLabel.getProductFileName() + " - %s", analyticsAction);
```

Comment:
Nit from last time I didn't spot, this would be clearer as `String.format("%s - %s", analyticsLabel.getProductFileName(), analyticsAction)` otherwise your combining string contacting with `String.format` which is unnecessary.